1. CWE Base Mod/Cold War Era Mod (CWE)/common/history/buildings/usa.txt

```diff
@@ -1,262 +1,7 @@
-		}
-	}
-	s:STATE_WISCONSIN = {
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_INDIANA = {
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_MICHIGAN = {
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_OHIO = {
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_KENTUCKY={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_TENNESSEE={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_MISSISSIPPI={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_ALABAMA={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_FLORIDA={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_GEORGIA={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_WEST_VIRGINIA={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_MARYLAND={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_DELAWARE={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_PENNSYLVANIA={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_NEW_JERSEY={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_ILLINOIS={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_RHODE_ISLAND={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_CONNECTICUT={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_MASSACHUSETTS={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_NEW_HAMPSHIRE={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_VERMONT={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_MAINE={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_NORTH_CAROLINA={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_UTAH={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_SOUTH_CAROLINA={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_HAWAIIAN_ISLANDS={
-		region_state:USA = {
-			effect_starting_buildings_2 = yes
-		}
-	}
-	s:STATE_TEXAS={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_NEVADA={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_ARIZONA={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_NEW_MEXICO={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_CALIFORNIA={
-		region_state:USA = {
-			effect_starting_buildings_4 = yes
-		}
-	}
-	s:STATE_SONORA={
-		region_state:USA = {
+	c:USA = {
+		every_scope_state = {
```

Comment:
Only incorporated states should have the level 4 buildings. Unincorporated states (like Alaska and Hawaii and the US mandates in Micronesia) should have level 1 or 2 buildings instead.

Reply:
Commit 9d49a0a solves this